### ISSUE TITLE
Applying to get one of my newest lists trusted in Nano

### ISSUE THREAD

**Issue by DandelionSprout** [NanoAdblocker/NanoCore#266]
Related to https://github.com/NanoAdblocker/NanoCore/issues/231, I have now after 6 months promoted that particular list of mine from Sandbox status to fulltime list status. Thus I hereby request for https://raw.githubusercontent.com/DandelionSprout/adfilt/master/IDCACnano.txt to become trusted by default in Nano Adblocker.

If accepted, this should probably also apply to its mirror links as well ([GitLab](IDCACnano.txt), [Repo.or.cz](https://repo.or.cz/FilterMirrorRepo.git/blob_plain/refs/heads/master:/IDCACnano.txt)).

**Comment by DandelionSprout** [comment]
If needed for context, then here's the file from *I Don't Care About Cookies*'s Firefox extension version that I took the entries from (although I had to change the file format from JSON to .txt): [common6.txt](https://github.com/NanoAdblocker/NanoCore/files/3180733/common6.txt)

**Comment by jspenguin2017** [comment]
Hi, privileged filter lists must be served from NanoFilters repository. Please, create a new folder named `ThirdPartyPrivileged` and send a PRs to NanoFilters to register your privileged filter list. 

Also, keep in mind that script snippets don't run on true `document-start`, so simply converting the rules may not be enough.

**Comment by DandelionSprout** [comment]
I was able to get the list to work on `archiveofourown` and `Twitter`, and I have reason to assume that most of the non-functional entries are a result of how IDCAC's maintainer tries to waddle his way through a five-digit amount of user reports about unblocked cookie notifications, thus he rarely gets time to polish his existing entries.

I'll make a pull request any moment now.

**Comment by jspenguin2017** [comment]
Alright, closing this in favor of the PR.
